flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/KeyedProcessOperatorTest.java

```diff
@@ -50,6 +52,30 @@
 	@Rule
 	public ExpectedException expectedException = ExpectedException.none();
 
+	@Test
+	public void testKeyQuerying() throws Exception {
+
+		KeyedProcessOperator<Integer, Tuple2<Integer, String>, String> operator =
```

Comment:
The `OneInputStreamOperatorTestHarness` is `AutoCloseable` so I would recommend to go with
```
try(harness=...) {
    ...
}
```
And remove the explicit call to `harness.close()`. This is a nice practice to start enforcing in new tests as it cleans up any leaks in case of exceptions and stuff.